### DISCUSSION TITLE
So lets talk about BBQ Sauce.

### DISCUSSION THREAD

**Discussion by ArcaneMusic** [space-wizards/space-station-14#3988] [General]
Hello Space Station 13, my name is ArcaneMusic, a contributor over at /tg/station on the SS13 development branch, as well as a spriter, admin, and general cool dude.

[We need to talk about BBQ sauce.](https://github.com/tgstation/tgstation/pull/46898)

When I PR'd this originally, is spawned a long, arduous conversation about _HOW_ you should be able to make BBQ sauce. Everyone has a way of doing BBQ sauce. Some people DEMAND vinegar. Others SWEAR by adding tomato paste. Disgusting BBQ Perverts suggested adding Mayo to your BBQ sauce.

Seeing as ya'll have ported over ribs, I suppose it should be brought up.

**Comment by gradientvera** [comment]
This is very important.

**Comment by SweptWasTaken** [comment]
Well since I was the one who ported over ribs and bbq sauce to SS14, i'd say mixing Ketchup and Vinegar together should give you BBQ sauce.

**Comment by moonheart08** [comment]
why not just add multiple valid recipes?
